.travis.yml

```diff
@@ -0,0 +1,26 @@
+sudo: required
+language: node_js
+node_js: 8.0
+
+cache: npm
+
+before_install: npm i -g npm@6.5.0
+install:
+  - npm ci
+  - npm i polymer-cli
+
+before_script:
+  - tslint -c ./tslint.json --project ./
```

Comment:
npm run test

Reply:
it is already added in script section:
script:
  - xvfb-run npm run test

Reply:
I added npm run build